test/e2e/specs/blocks/__snapshots__/quote.test.js.snap

```diff
@@ -58,11 +58,7 @@ exports[`Quote can be converted to paragraphs and renders a paragraph for the ci
 <!-- /wp:paragraph -->"
 `;
 
-exports[`Quote can be converted to paragraphs and renders a void paragraph if both the cite and quote are void 1`] = `
-"<!-- wp:paragraph -->
-<p></p>
-<!-- /wp:paragraph -->"
-`;
+exports[`Quote can be converted to paragraphs and renders a void paragraph if both the cite and quote are void 1`] = `""`;
```

Comment:
I saw this change in another pull request. What's going on with this? It doesn't seem like we should be expecting any changes here.

Reply:
See https://github.com/WordPress/gutenberg/pull/11595#discussion_r232256521.